### DISCUSSION TITLE
Builds time out

### DISCUSSION THREAD

**Discussion by markbreuss** [codemagic-ci-cd/codemagic-docs#2577] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

My Builds currently all time out.
Usually I would just try it again tomorrow and call it a day.

But I have to ship a hotfix to our beta customers - and did not get back an ETA form support.

I know how stressed you guys are at the moment but I would stll apprecieate some info so I can inform our customers.

Thanks!

### Expected results

ETA for a fix

### Actual results

Flutter download Timeout

### Build id (optional)

665f283075ec3417de0825f1

**Comment by jsf-ns** [comment]
Same issue here. It has been happening since last week btw

**Comment by alih552** [comment]
same issue here

**Comment by momer** [comment]
Likewise, my build is timing out as well; ref: 665f35abc99a44cdb84f10ec

**Comment by kieranmccwayleadr** [comment]
Lads, to stop the Me too comments like last time, check the status page. It's degraded https://status.codemagic.io/

**Comment by myfknoll** [comment]
Same here with flutter: 3.22.1

**Comment by mikhail-tokarev** [comment]
Hi all! We are experiencing issues with network in one of our data center. We'll post more information once we have on https://status.codemagic.io and https://github.com/orgs/codemagic-ci-cd/discussions/2568 

I'm going to close this issue to keep all communication in one place. Thank you for your understanding!
